### PR TITLE
Bicycle Jouster

### PR DESCRIPTION
**Pull request by Gabe-Lincoln**
Adds the bicycle Jouster profession. I think this profession should be balanced at 3 points, but if anyone has any concerns, I'd like to hear them. I also think it'd be a fun profession to play.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -5324,5 +5324,49 @@
                 "panties"
             ]
         }
+    },   
+    {
+        "type": "profession",
+        "ident": "cyclist_joust",
+        "name": "Bicycle Jouster",
+        "description": "A vagrant before the cataclysm, you had settled down in an underground jousting ring. Now, that ring is full of undead.",
```
Comment (Night-Pryanik) [inline data/json/professions.json]: Two spaces after period between sentences.

**Comment by Night-Pryanik** [comment]
Three points is too much for a low-level skills (especially not so important like driving) and a set of not so cool gear. I'd say it worth two points.
Also, I don't see any backpack or similar gear. Unless I'm wrong, folding bike will be dropped on the ground after the first turn.

File: data/json/professions.json (hunk 1 above)
Comment (Gabe-Lincoln) [reply data/json/professions.json]: Thanks

**Comment by Gabe-Lincoln** [comment]
It's generally better than the Professional Cyclist trait that I copied it from, with a good weapon to start with, and some okay armor, and that is 3 points. However, there are some other good professions at 3 points, so I thought it was okay being at 3 points.

**Comment by cosmiccoincidence** [comment]
My main concern (although I think it sounds fun) is that most of the preexisting professions are things that would otherwise exist before the cataclysm (firefighter, biker, cyclist, lawyer, backpacker, crackhead, etc.) and this seems like something that wouldn't exist until after the cataclysm.
Maybe that's just my perspective though.

**Comment by Chezzo** [comment]
This profession has the same skills and the same items as Professional Cyclist, adds three stabbing and one mechanics, and adds motorcycle armor (Cyclist doesn't come with armor) and a better helmet for no additional points?

**Comment by Gabe-Lincoln** [comment]
I think Professional Cyclist is a bit overcosted, but I can definitely see why it should be 4 points. My first instinct was actually to put it at 5 points. I just don't have a good enough grasp of how point costs are laid out to be sure where to put it.

**Comment by Xhuis** [comment]
Sounds interesting enough, though it could use a unique profession trait to give it a reason to exist. Perhaps it could make attacking from vehicles have a higher to-hit, Then again, this appears to be a webeditor PR, so I'm not expecting a bunch of c++ out of the gate.

File: data/json/professions.json [hunk 2]
```diff
@@ -5324,5 +5324,49 @@
...
+        "points": 4,
+        "skills": [
+            {
+                "level": 3,
+                "name": "driving"
+            },
+            {
+                "level": 3,
+                "name": "stabbing"
+            }
```
Comment (OrenAudeles) [inline data/json/professions.json]: missing separator after "stabbing" skill entry, and after "dodge" skill entry. Causing errors in build systems.

File: data/json/professions.json [hunk 3]
```diff
@@ -5324,5 +5324,49 @@
...
+                "name": "driving"
+            },
+            {
+                "level": 3,
+                "name": "stabbing"
+            },
+	        {
+                "level": 2,
+                "name": "dodge"
+            }
```
Comment (OrenAudeles) [inline data/json/professions.json]: Still need a comma here after the "dodge" skill entry.

**Comment by ZhilkinSerg** [comment]
Suggestion:

Let's say Bionic Jousting it is some kind of sport.

Add CBM head and chest armor plating instead of helmet and armor, name it "Bionic Jouster" and put profession to HTLL start (maybe also add some malfunctioning bionics and no bike).

**Comment by Gabe-Lincoln** [comment]
The entire point of the profession was that it was a silly thing because you can use a melee weapon on a bike. Removing the bike isn't an option.

**Comment by ZhilkinSerg** [comment]
Pleas resolved conflicts.

> The entire point of the profession was that it was a silly thing because you can use a melee weapon on a bike. Removing the bike isn't an option.

Silly things belong to Crazy Cataclysm. Did you consider moving new class there?

**Comment by ZhilkinSerg** [comment]
Marking as Stalled to clean up PR queue. Please ping me to get it reopened when you feel you can continue working on it.

**Pull request closed without merging**